src/calculateRentalCost.js

```diff
@@ -1,10 +1,27 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const Price = 40;
+  const ShortTerm = 3;
+  const ShortDiscount = 20;
+  const LongTerm = 7;
+  const LongDiscount = 50;
+  const TotalPrice = Price * days;
```

Comment:
The name 'TotalPrice' could be misleading since it's actually the price before any discounts are applied. Consider renaming it to 'basePrice' or 'priceBeforeDiscount' for clarity.